support/COMMUNITY_CONTACTS.md

```diff
@@ -0,0 +1,78 @@
+# Duty
+
+Every week we dedicate one individual (the community contact) to watch for user
+issues and either answer them or redirect the questions to other who can. The
+community contact's duty (subject to change) is as followed:
+
+## Daily check list
+
+- Check the knative-users@ mailing list and #eventing-questions Slack channel for
+  new unanswered questions.
+- Check
```

Comment:
From the discussion here https://knative.slack.com/archives/C9JP909F0/p1596699092240700 how do we ensure that everybody in this roster has eventing triage permissions? Should all of them add themselves to the group?

Reply:
The Eventing Triage group is intended to be a stopgap until an official triage permission mechanism is defined. If the introduction of the community contact rotation effectively means all approvers need to have triage permissions, I think it bumps up the priority of that discussion.

Reply:
I think since it is a rotation, we should only need to add ACLs during the week of duty. Right now that that is a manual step, and should act as a handoff / reminder mechanism for the next individual on the rotation but when the process becomes more established we should be able to automate much.

Reply:
I've updated the instruction so that the triage permissions are only added for active individual. This keeps the rotation setup independent from the official triage permission mechanism conversation. PTAL

Reply:
@tcnghia i think there is no need to rotate these ACLs. People that are part of community contacts already check issues, solve them, so IMO we should grant permanently these ACL.

Reply:
@slinkydeveloper while I think that could be a good decision, if there is already another discussion regarding that I don't want to make the decision for that discussion in this PR. Once the Eventing folks make a decision on that thread we can always update this instruction.

@grantr @slinkydeveloper  WDYT?

Reply:
I'm out all week next week, so don't block this PR on me responding here. I'm personally fine with adding any eventing approver to the triage team, as long as its understood that this arrangement is provisional and likely to be replaced by something else.

Reply:
+1 in that case I'll update the doc to reflect what @slinkydeveloper suggested. This is a new process and can/should be modified as we start using it.

Reply:
Done